packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -32,10 +32,25 @@ interface DeviceSettingsDropdownProps {
   disabled?: boolean;
 }
 
+const resetOptionsIOS: Array<{ label: string; value: AppPermissionType; icon: string }> = [
+  { label: "Reset All Permissions", value: "all", icon: "check-all" },
+  { label: "Reset Location", value: "location", icon: "location" },
+  { label: "Reset Photos", value: "photos", icon: "file-media" },
+  { label: "Reset Contacts", value: "contacts", icon: "organization" },
+  { label: "Reset Calendar", value: "calendar", icon: "calendar" },
+];
+
+const resetOptionsAndroid: Array<{ label: string; value: AppPermissionType; icon: string }> = [
+  { label: "Reset All Permissions", value: "all", icon: "check-all" },
+];
+
 function DeviceSettingsDropdown({ children, disabled }: DeviceSettingsDropdownProps) {
-  const { project, deviceSettings } = useProject();
+  const { project, deviceSettings, projectState } = useProject();
   const { openModal } = useModal();
 
+  const resetOptions =
+    projectState.selectedDevice?.platform === "iOS" ? resetOptionsIOS : resetOptionsAndroid;
```

Comment:
If we expect device to be selected here, you can use `projectState.selectedDevice!.platform` to remove possibly undefined result.

Reply:
we don't. the menu is active regardless of whether device is selected. This can only happen for a very short time and in that case I don't think anyone will expect the reset options to do anything. I don't feel like adding extra code here to handle such case